debian/rules.in

```diff
@@ -9,7 +9,7 @@
 # Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
-CUDA_OPT = $(shell echo ${DEB_BUILD_PROFILES} | grep -qw cuda \
+CUDA_OPT = $(shell echo ${PATH} | grep -qw cuda \
```

Comment:
why checking PATH var?
maybe check if /usr/local/cuda exists?

Reply:
How about the CUDA_VERSION env var?
We can check if the version is new enough.